quantities/units/Imperial/Volume/Gallon.cs

```diff
@@ -13,7 +13,7 @@ namespace Quantities.Units.Imperial.Volume;
     static T IInjectUnit<ILength>.Inject<T>(in Creator<ILength, T> inject, in Double self)
     {
         const Double galToCuFt = ToCuMetre / (Foot.ToMetre * Foot.ToMetre * Foot.ToMetre);
-        return inject.Other<Foot>(galToCuFt * self);
+        return inject.Imperial<Foot>(galToCuFt * self);
```

Comment:
Could you rename `galToCuFt` to _galleonToCubicFeet_?

Reply:
Implemented